### DISCUSSION TITLE
[Idea]: Full list of paragraphs for each WT image

### DISCUSSION THREAD

**Discussion by ux-git** [sircharlo/meeting-media-manager#2405] [Ideas] [answered]
### Is your idea related to a problem?

Currently, each image from Watchtower articles has only one paragraph displayed in M³. Though it's useful, brothers still need to not rely on it and always double-check the article for a full list of paragraphs related to a particular image.

### Describe your idea

To include all the paragraphs mentioned in each article, in the description right under each image

![Screenshot_20240309-190626](https://github.com/sircharlo/meeting-media-manager/assets/80993061/dbb6b54f-b5bf-4aab-a083-b73b2f63c6a0)

### Describe alternatives you've considered

To always pay more attention to the article.

**Comment by mtdvlpr** [comment, marked as answer]
Implemented as of `v24.10.6`
